frontend/src/slices/auth/auth.slice.ts

```diff
@@ -1,16 +1,20 @@
 import { createSlice } from '@reduxjs/toolkit';
+import { type UserSignUpResponseDto } from 'shared/build/index.js';
 
 import { DataStatus } from '~/libs/enums/enums.js';
+import { storage, StorageKey } from '~/libs/packages/storage/storage.js';
 import { type ValueOf } from '~/libs/types/types.js';
 
 import { signUp } from './actions.js';
 
 type State = {
   dataStatus: ValueOf<typeof DataStatus>;
+  user: UserSignUpResponseDto;
 };
 
 const initialState: State = {
   dataStatus: DataStatus.IDLE,
+  user: { token: '', user: { id: 1, email: '' } },
```

Comment:
```suggestion
  user: null,
```
default should be null